draft-ietf-rtcweb-jsep.xml

```diff
@@ -2436,8 +2442,9 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             <t>The "a=rtcp-mux" line MUST only be added if present in
             the most recent answer.</t>
 
-            <t>The "a=rtcp-mux-only" line MUST only be added if present
-            in the most recent answer.</t>
+            <t>If the RTP/RTCP multiplexing policy is "require",
```

Comment:
The document actually says you use if if you *negotiated* mux-exclusive.

https://tools.ietf.org/html/draft-ietf-mmusic-mux-exclusive-10#section-4.5